shared_model/interfaces/common_objects/signable_hash.hpp

```diff
@@ -54,8 +54,6 @@ namespace shared_model {
      * limitations: it requires to have write access for elements for some
      * internal operations.
      */
-    using SignatureSetType =
```

Comment:
I think `SignatureSetType` should be in one file with `class SignableHash`. So, `class SignableHash` should be moved inside 'class Signable` as a part of it's implementation or `SignatureSetType` should be moved here.

Reply:
Yes please, otherwise `SignatureSetType` will not be accessible easily from outer scope because `Signable` is a template class.